completion_aggregator/api/v1/views.py

```diff
@@ -416,6 +426,28 @@ def get(self, request, course_key):
         )
         return paginator.get_paginated_response(serializer.data)
 
+    def get(self, request, course_key):
+        """
+        Handler for GET requests.
+        """
+        params = {}
+        if request.query_params.get('user_ids', None):
+            params['user_ids'] = (int(id) for id in re.split(r',|\.', request.query_params['user_ids']))
```

Comment:
We could simplify this regexp to `[,.]`. It will perform around 2x less steps this way.

Reply:
Woah! I've done some testing here and it's faster indeed. I'll make the changes. 
I'm not very experienced at regex so I went with what worked, but I'll read a little more about it.

Reply:
I personally liked [these](https://www.hackerrank.com/domains/regex) challenges for practicing `/regexp?/` :)